internal/controllers/topology/cluster/patches/engine.go

```diff
@@ -302,29 +303,29 @@ func applyPatchesToRequest(ctx context.Context, req *runtimehooksv1.GeneratePatc
 			log.V(5).Infof("Accumulating JSON patch: %s", string(patch.Patch))
 			jsonPatch, err := jsonpatch.DecodePatch(patch.Patch)
 			if err != nil {
-				return errors.Wrapf(err, "failed to apply patch with uid %q: error decoding json patch (RFC6902): %s",
-					requestItem.UID, string(patch.Patch))
+				log.V(5).Infof("Failed to apply patch with uid %q: error decoding json patch (RFC6902): %s: %s", requestItem.UID, string(patch.Patch), err)
+				return errors.Wrap(err, "failed to apply patch: error decoding json patch (RFC6902)")
```

Comment:
I think it's much more constant than the uid - we can't assume it's constant, but it won't change on every reconcile in a normal case I think. It could change in some case, though so it depends on how cautious this should be.